### PR TITLE
Build flutter_gallery with bitcode

### PR DESCRIPTION
**Pull request by dnfield**
Updates flutter_gallery to build with bitcode.

Locally, built with app thining and bitcode enabled, results in a 39mb IPA.  I'm not sure if or how this will reflect in the benchmark.

### CODE REVIEW

File: examples/flutter_gallery/ios/Runner.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -183,6 +183,7 @@
 				TargetAttributes = {
 					97C146ED1CF9000F007C117D = {
 						CreatedOnToolsVersion = 7.3.1;
+						DevelopmentTeam = S8QB4VV633;
```
Comment (xster) [inline examples/flutter_gallery/ios/Runner.xcodeproj/project.pbxproj]: revert

**Comment by xster** [comment]
RSLGTM

File: examples/flutter_gallery/ios/Runner.xcodeproj/project.pbxproj (hunk 1 above)
Comment (dnfield) [reply examples/flutter_gallery/ios/Runner.xcodeproj/project.pbxproj]: You're too fast! I was force pushing a change up to hide my shame!

Comment (xster) [reply examples/flutter_gallery/ios/Runner.xcodeproj/project.pbxproj]: giphy-10

**Comment by dnfield** [comment]
This will need a version bumpfor the Xcode on Cirrus.

**Pull request merged**